scripts/update_clocks.py

```diff
@@ -0,0 +1,187 @@
+        if num_of_retries == 10:
+            raise ValueError("Connection to ES failed!")
+        num_of_retries += 1
+        time.sleep(30)
+
+    return es
+
+async def upload_release_asset(session, token, repo, tag_name, asset_path, delete_existing=True):
+    return 0
+    """ Upload a release asset to GitHub """
+    print(f"Preparing to upload asset: {os.path.basename(asset_path)}...")
+    headers = {'Authorization': f'token {token}', 'Content-Type': 'application/octet-stream'}
+
+    release_url = f"https://api.github.com/repos/{repo}/releases/tags/{tag_name}"
+    if 'latest' == tag_name:
+        release_url = f"https://api.github.com/repos/{repo}/releases/latest"
+
+    async with session.get(release_url, headers=headers) as response:
+        response_data = await response.json()
+        release_id = response_data['id']
+
+    # Handle pagination to get all assets
+    page = 1
+    asset_deleted = False
+    asset_name = os.path.basename(asset_path)
+    while True:
+        if asset_deleted:
+            break
+        url = f'https://api.github.com/repos/{repo}/releases/{release_id}/assets?page={page}&per_page=30'
+        response = requests.get(url, headers=headers)
+        response.raise_for_status()
+        assets = response.json()
+
+        # If no more assets, break out of loop
+        if not assets:
+            break
+
+        # Check if the asset already exists
+        for asset in assets:
+            if asset['name'] == asset_name:
+                # If the asset exists, delete it
+                delete_url = asset['url']
+                if delete_existing:
+                    print(f'Deleting existing asset: {asset_name}')
+                    delete_response = requests.delete(delete_url, headers=headers)
+                    delete_response.raise_for_status()
+                    print(f'Asset deleted: {asset_name}')
+                asset_deleted = True
+                break
+
+        page += 1
+
+    upload_url = f"https://uploads.github.com/repos/{repo}/releases/{release_id}/assets?name={os.path.basename(asset_path)}"
+    async with aiofiles.open(asset_path, 'rb') as f:
+        data = await f.read()
+    async with session.post(upload_url, headers=headers, data=data) as response:
+        result = await response.json()
+    print(f"Upload completed for: {os.path.basename(asset_path)}.")
+    return result
+
+async def main(token, repo, tag_name):
+    # Generate clocks.json
+    input_directory = "./"
+    output_file = "./output/docs/clocks.json"
+    clocksGenerator = GenerateClocks(input_directory, output_file)
+    clocksGenerator.generate()
+    async with aiohttp.ClientSession() as session:
+        upload_result = await upload_release_asset(session, token, repo, tag_name, output_file)
+
+    es = initialize_es()
+
+    release_details = fetch_release_details(args.repo, args.token, args.tag_name)
+
+    version = increment_version(release_details['tag_name'].replace('v', ''))
```

Comment:
Change the version to be incremented from ES instead of current TAG.

Currently, it will always have the same version.